src/main.ts

```diff
@@ -42,10 +42,10 @@ const MODE_TYPES: Array<string> = Object.values(MODES)
 const DEFAULT_CONTROL = [MODES.HVAC, MODES.PRESET]
 
 const ICONS = {
-  UP: 'hass:chevron-up',
```

Comment:
`chevron-up` is still available under `mdi` icons: https://icon-sets.iconify.design/mdi/chevron-up/ Any reason you changed it to a different icon?